defold-rive/commonsrc/rive_ddf.proto

```diff
@@ -31,6 +64,10 @@ message RiveModelDesc
     optional bool create_go_bones           = 6 [default=false];
     optional string artboard                = 7;
 
+    optional CoordinateSystem coordinate_system = 8 [default = COORDINATE_SYSTEM_FULLSCREEN];
```

Comment:
Is this a good name for the setup? Basically what I want to achieve is to be able to position things together with defold content in the same coordinate system as well as position things according to the .riv files themselves, using the alignment options that are available in the Rive API. Not sure the naming here is good thoug, should it be something like "Game" / "GUI"? "Game" / "Rive Alignment"? 🤷